app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -191,21 +192,31 @@ class SamplesView extends React.Component {
         ? mapSidebarSelectedSampleIds
         : selectedSampleIds;
 
-    const log = () =>
-      logAnalyticsEvent("SamplesView_heatmap-icon_clicked", {
-        selectedSampleIds: targetSampleIds.length,
+    const heatmapOptions = [
+      { text: "Taxon Heatmap", value: "/visualizations/heatmap" },
+    ];
+    if (this.props.admin) {
+      heatmapOptions.push({
+        text: "AMR Heatmap",
+        value: "/amr_heatmap",
       });
+    }
     return targetSampleIds.size < 2 ? (
       <HeatmapIcon className={cx(cs.icon, cs.disabled, cs.heatmap)} />
     ) : (
-      <a
-        onClick={log}
-        href={`/visualizations/heatmap?sampleIds=${Array.from(
-          targetSampleIds
-        )}`}
-      >
-        <HeatmapIcon className={cx(cs.icon, cs.heatmap)} />
-      </a>
+      <HeatmapIconDropdown
```

Comment:
It doesn't feel necessary to pull this out into a separate component. (I'm not sure why `DownloadIconDropdown` is a separate component.) We would want to create a separate component is if there is a substantial amount of logic in the component and we expect to use the component across multiple pages. Neither seems to apply here.